src/components/ui/carousel.tsx

```
@@ -1,262 +1,212 @@
+            ref={ref}
+            role="group"
+            aria-roledescription="slide"
+            className={cn("min-w-0 shrink-0 grow-0 basis-full", orientation === "horizontal" ? "pl-4" : "pt-4", className)}
+            {...props}
+        />
+    );
+});
+CarouselItem.displayName = "CarouselItem";
+
+const CarouselPrevious = React.forwardRef<HTMLButtonElement, React.ComponentProps<typeof Button>>(
+    ({ className, variant = "outline", size = "icon", ...props }, ref) => {
```

ESLint

[error] 'variant' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)